docs/learning/ml5-glossary.md

```diff
@@ -43,7 +43,7 @@ const classifier = ml5.imageClassifier("MobileNet", (err, model) => {
 });
 
 // Make a prediction with the selected image and pass a callback function with two arguments
-classifier.predict(image, (err, results) => {
+classifier.classify(image, (err, results) => {
```

Comment:
A couple notes here!

1. I believe we've changed everything to have the `results` be the first argument and the `error` is optional / mostly ignore by all examples for simplicity.
2. We may want to avoid the `=>` syntax as it's not the usual beginner-friendly p5.js style and not typically used in the examples. That said if there is a reason to include that here in the glossary I think it's ok!!

Reply:
Got it! I will refer to the examples and adjust the content accordingly.

Reply:
@shiffman @MOQN @QuinnHe In addition to what you mentioned, do we need to keep the "promise" part in the glossary? I think the way to explain it in the glossary right now is to try to use it as an alternative way of callback functions. But I checked all the examples, we have no example written this way. The example code also uses arrow function... 
![CleanShot 2024-06-17 at 17 22 43@2x](https://github.com/ml5js/ml5-website-v02-docsify/assets/10086000/a4a4ddbb-fd3f-4441-a05f-9ef9853ddfe3)

Reply:
Great question! I might be inclined to take it out. @ziyuan-linn do you know if we are continuing to even support promises with the new API? We could make an FAQ page or some type of learning page about callbacks and promises, but this is likely out of scope for what we can maintain right now! I'm ok either way!

Reply:
Factory functions like `ml5.imageClassifier` no longer return a promise since it has to return the `ImageClassifier` object itself to support p5.js `preload`, so at least this specific example should be taken out. The `detectStart` function also does not support promise since the callback will be called repeatedly. 

I believe everything else in the new API still supports promise, though it might be less confusing if we stick to callback only for the examples.